li_create_accept_policy

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Layered Insight - All Rights Reserved
+# Unauthorized copying of this file, via any medium is strictly prohibited
+# Proprietary and confidential
+
```

Comment:
Think our users could benefit from a comment in here (and in other scripts) that tells a bit more about what the script does?